pages/1.12/cli/configure/index.md

```diff
@@ -32,3 +32,70 @@ To configure a proxy for the CLI:
 *   Define `no_proxy` for domains that you don’t want to use the proxy for:
 
         export no_proxy=".mesos,.thisdcos.directory,.dcos.directory,.zk,127.0.0.1,localhost,foo.bar.com,.baz.com”
+
+# Environment variables
+
+The DC/OS CLI supports the following environment variables, which can be set dynamically.
+
+<a name="dcos-cluster"></a>
+#### `DCOS_CLUSTER` (DC/OS CLI 0.5.x and higher only)
+
+The [attached](/1.10/cli/command-reference/dcos-cluster/dcos-cluster-attach/) cluster. To set the attached cluster, set the variable with the command:
+
+```bash
+export DCOS_CLUSTER=<cluster_name>
+```
+
+<a name="dcos-config"></a>
+#### `DCOS_CONFIG` (DC/OS CLI 0.4.x only)
+
+The path to a DC/OS configuration file. If you put the DC/OS configuration file in `/home/jdoe/config/dcos.toml`, set the variable with the command:
+
+```bash
+export DCOS_CONFIG=/home/jdoe/config/dcos.toml
+```
+
+If you have the `DCOS_CONFIG` environment variable configured:
+
+- After conversion to the [new configuration structure](#configuration-files), `DCOS_CONFIG` is no longer honored.
+- Before you call `dcos cluster setup`, you can change the configuration pointed to by `DCOS_CONFIG` using `dcos config set`. This command prints a warning message saying the command is deprecated and recommends using `dcos cluster setup`.
+
+
+<a name="dcos-dir"></a>
+#### `DCOS_DIR` (DC/OS CLI 0.5.x and higher only)
+
+The path to a DC/OS configuration directory. If you want the DC/OS configuration directory to be `/home/jdoe/config`, set the variable with the command:
+
+```bash
+export DCOS_DIR=/home/jdoe/config
+```
+
+1. Optionally set `DCOS_DIR` and run `dcos cluster setup` command.
+
+    ```
+    export DCOS_DIR=<path/to/config_dir> (optional, default when not set is ~/.dcos)
+    dcos cluster setup <url>
+    ```
+
+   This setting generates and updates per cluster configuration under `$DCOS_DIR/clusters/<cluster_id>`. Sets newly set up cluster as the attached one.
```

Comment:
This line "Sets newly set up cluster as the attached one." is not clear to me. I do not understand or see what "the attached one" links/points/refers to on this page.

Did you mean to say "Generates a newly setup cluster as seen here."
If yes, please add a link to the setup cluster.